cms/templates/cms/home_page.html

```diff
@@ -75,7 +75,46 @@
         </div>
       </div>
 
-            <section class="row current-programs-section">
+      <div class="row how-it-works-section">
+
+      <div class="centered-content">
+
+        <h3>How It Works</h3>
+        <div class="hiw col-md-3">
+          <div class="reasons-images-wrap">
+             <img src="{% static 'images/hiw1.png' %}" alt="">
+          </div>
+
+          <h5>Online Learning</h5>
```

Comment:
"How It Works" is in a `<h3>`, so the next heading level should be `<h4>`. Please adjust all the headings in this section.